doc/classes/Input.xml

```diff
@@ -117,6 +165,26 @@
 				Returns the current value of the joypad axis at index [param axis].
 			</description>
 		</method>
+		<method name="get_joy_gravity" qualifiers="const">
+			<return type="Vector3" />
+			<param index="0" name="device" type="int" />
+			<description>
+				Returns the requested joypad's gravity value in m/s² using accelerometer readings.
+				If the joypad doesn't have an accelerometer sensor or it's currently disabled, this method returns [constant Vector3.ZERO].
+				For a joypad held in front of you, the returned axes are defined as follows:
+				-X ... +X : left ... right;
+				-Y ... +Y : bottom ... top;
+				-Z ... +Z : farther ... closer.
+				For the joypad to be able to send accelerometer (and gravity) data, the sensor must exist on the joypad and it must be enabled using [method enable_joy_accelerometer].
+				The raw accelerometer readings can be retrieved using [method get_joy_accelerometer].
+				See [method start_joy_motion_calibration] for an example on how to use joypad motion sensors and calibration in your games.
+				[b]Note:[/b] It's recommended to disable the motion sensors after they're no longer used, because otherwise it might drain the controller battery faster.
```

Comment:
```suggestion
				[b]Note:[/b] It's recommended to disable the motion sensors when they're no longer being used, because otherwise it might drain the controller battery faster.
```